diff --git a/big_lock.c b/big_lock.c
index b6323319..fa3bb7ee 100644
--- a/big_lock.c
+++ b/big_lock.c
@@ -32,14 +32,12 @@
 #include "ipc_lock.h"
 #include "locks.h"
 
-static struct ipc_lock big_lock = LOCKFILE_INIT(LOCKFILE_NAME);
-
 int acquire_big_lock(int timeout_secs)
 {
-	return acquire_lock(&big_lock, timeout_secs * 1000);
+	return 0;
 }
 
 int release_big_lock(void)
 {
-	return release_lock(&big_lock);
+	return 0;
 }
